sass/base/_components.scss

```diff
@@ -48,6 +48,7 @@
 @import "components/promo-header/promo-header";
 @import "components/quote-block/content-wall-quote";
 @import "components/related-articles/related-articles";
+@import "components/schools-look-up/schools-look-up";
```

Comment:
Again, do we need this as part of this PR?

Reply:
that was a merge from master that had a little hiccup and I had to commit myself